worker/download.go

```diff
@@ -339,6 +344,8 @@ outer:
 			if resp.err != nil {
 				mgr.logger.Errorf("download slab %v failed: %v", resp.index, resp.err)
 				return resp.err
+			} else if resp.overpaid {
+				mgr.logger.Debugf("download slab %v succeeded by overpaying", resp.index)
```

Comment:
I'd make this a WARN maybe and maybe change the msg to something like `download for slab %v had to overpay to succeed`.